about.html

```diff
@@ -0,0 +1,195 @@
+            </a>
+            <ul class="navbar-nav ml-auto">
+              <li class="nav-item">
+                <a class="nav-link pr-5" href="index.html">
+                  Home
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link pr-5" href="about.html">
+                  About
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link pr-5" href="#">
+                  Join
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link pr-5" href="#">
+                  Sponsor
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link pr-5" href="#">
+                  News
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4 campaign">
+                <a class="nav-link orange pr-5" href="#">
+                  CM Campaign
+                </a>
+              </li>
+            </ul>
+        </nav>
+        </div>
+        <nav class="toggle d-md-none d-xs-block">
+        <img src="assets/images/menu.png" alt="hamburger menu" class="hamburger_menu_icon" />
+        <ul class="mobile_side_nav">
+          <li class="menu_link closebtn"><a href="javascript:void(0)">&times;</a></li>
+          <li class="menu_link"><a href="index.html" class="menulink">Home</a><hr class="section_title_hr"/></li>
+          <li class="menu_link"><a href="about.html" class="menulink">About</a><hr class="section_title_hr"/></li>
+          <li class="menu_link"><a href="#">Join</a><hr class="section_title_hr"/></li>
+          <li class="menu_link"><a href="#">Sponsor</a><hr class="section_title_hr"/></li>
+          <li class="menu_link"><a href="#">News</a><hr class="section_title_hr"/></li>
+        </ul>
+        </nav>`
+    </header>
+    <main class="parent_main">
+        <section class="about_header">
+          <div class="welcome-text col-12 pt-5 mt-5">
+            <h3 class="text-left text-center hello_intro orange">Tokyo's Nightmare</h3>
+            <h1 class="text-uppercase text-center text-md-center font-weight-bold text-img-effect">CRIMSON DAWN <BR> CAR MEETING<BR> 2022</h1>
+          </div>
+          <div class="about_desc col-12 mt-4 mb-4">
+            <p class="bg-white text-center p-4">The most known, most waited and BIGGEST car meeting is finally coming back after all the Cov-19 restrictions. It will bring back all the legendary tuners, the new-comers and the craziest tunes, all in one place.  </p>
+          </div>
+          <div class="event-date col-12 pb-3">
+            <p class="text-center w-50 m-auto pb-3">
+              Please contact us over email for any further questions
+              about Crimson Dawn Car Meeting.
+              <span class="font-weight-bold pb-4">
+                <u>carmeet@crimsondawn.com</u>
```

Comment:
- [ ] It would be better to align the email to the center of the page to improve your UI design.

<img width="205" alt="image" src="https://user-images.githubusercontent.com/77190203/157698308-494132b4-8a6e-48e8-84f4-7836ac7a5c66.png">